collector/src/main/java/com/bitmovin/analytics/BitmovinAnalytics.java

```diff
@@ -86,6 +92,31 @@ public void detachPlayer() {
         eventDataDispatcher.disable();
     }
 
+    private void addDeviceOrientationListener() {
+
+        SensorEventListener m_sensorEventListener = new SensorEventListener() {
```

Comment:
We don't use `_` in the variable names. We try to keep to the [google code style ](https://google.github.io/styleguide/javaguide.html)

A better name would just be `sensorEventListener`

Reply:
Also a good trick in IntelliJ to save on typing is to just select the expression that generates a variable and then use `Refactor -> Extract -> Variable` as IntelliJ will then infer the type and also suggest a usually semi-useful name. 

![image](https://user-images.githubusercontent.com/89680/67473442-f9310000-f652-11e9-96bf-0697f783b99e.png)

The shortut on my machine is `Option+Command+V`